engines/python/src/main/java/ai/djl/python/engine/PyPredictor.java

```diff
@@ -72,10 +72,11 @@ public List<O> batchPredict(List<I> inputs) throws TranslateException {
             int size = inputs.size();
             if (size == 1) {
                 Output output;
-                if (isRollingBatch) {
-                    output = rollingBatch.addInput((Input) first, timeout);
+                Input input = (Input) first;
+                if (isRollingBatch && !input.getProperties().containsKey("handler")) {
```

Comment:
trying to understand the logic here. Are we saying we are not supporting customer handler?